### PR TITLE
feat(giscus): use current language as fallback for `lang` param and add `loading` param

### PR DESCRIPTION
**Pull request by xiaoxiyao**
see #1062

### CODE REVIEW

File: layouts/partials/comments/provider/giscus.html [hunk 1]
```diff
@@ -1,3 +1,7 @@
+{{ $lang := .Language.LanguageCode }}
+{{ if not $lang }}
+    {{ $lang = .Language.Lang }}
+{{ end }}
```
Comment (CaiJimmy) [inline layouts/partials/comments/provider/giscus.html:4]: Hugo already does the fallback:

image

https://gohugo.io/methods/page/language/

So there's no need to check this in the template. Hence the code can be simplified, and changed to:

`data-lang="{{- default $.Language.LanguageCode .lang -}}"`

**Review by CaiJimmy — CHANGES_REQUESTED** [review]
Thanks!

File: layouts/partials/comments/provider/giscus.html (hunk 1 above)
Comment (xiaoxiyao) [reply layouts/partials/comments/provider/giscus.html]: Thanks! I've updated it.

**Review by CaiJimmy — APPROVED** [review]
Thanks!

**Pull request merged**